thriftpy/contrib/twitter_thrift/__init__.py

```diff
@@ -0,0 +1,124 @@
+"""
+Implementation of twitter thrift for thriftpy
+
+
+"""
+
+from __future__ import absolute_import
+
+import os.path
+import warnings
+from random import Random
+
+random = Random()
+
+from thriftpy.contrib.twitter_thrift.context import broadcast
+
+from thriftpy.protocol import TBinaryProtocolFactory
+from thriftpy.transport import TSocket, TSSLSocket, TFramedTransportFactory
+from ...thrift import TClient, TApplicationException, TMessageType, TType
+from ...parser import load
+
+# From:
+# https://github.com/twitter/finagle/blob/master/finagle-thrift/src/main/scala/com/twitter/finagle/thrift/ThriftTracing.scala
+trace_method = "__can__finagle__trace__v3__"
+trace_thrift = load(os.path.join(os.path.dirname(__file__), "tracing.thrift"))
```

Comment:
It scares me a little to have a filesystem call at the top level like this. If something goes wrong, this will fail at import time, which can be annoying to debug. Is there any way we could wrap this in an object or something and have it load lazily on first use?